CMakeLists.txt

```diff
@@ -426,6 +426,26 @@ if(NOT WIN32)
 endif()
 
 find_package(Readline REQUIRED)
+find_package(Augeas REQUIRED)
+
+if(LINUX)
+  find_package(LibSELinux REQUIRED)
+endif()
+
+if(APPLE)
+  if(NOT DEFINED ENV{BUILD_LINK_SHARED})
```

Comment:
I couldn't manage to tell CMake to do it without this logic. I'm far from a CMake expert though.